app/src/shared/components/DatePicker/index.jsx

```diff
@@ -61,7 +65,11 @@ class DatePicker extends React.Component<Props> {
                                     onChange={this.onChange}
                                     {...rest1}
                                 />
-                                <CalendarIcon />
+                                <CalendarIcon
```

Comment:
Here the icon is hugging the border a pixel or two too close :)

<img width="690" alt="Screen Shot 2019-11-20 at 16 08 53" src="https://user-images.githubusercontent.com/1064982/69225125-59fb1c00-0bb0-11ea-99d3-e2967ac56882.png">
Compare with: 
<img width="702" alt="Screen Shot 2019-11-20 at 16 11 07" src="https://user-images.githubusercontent.com/1064982/69225147-654e4780-0bb0-11ea-88d4-12778e58979b.png">